.github/ci_env.yml

```diff
@@ -0,0 +1,53 @@
+name: test
```

Comment:
This file is only necessary for your proposed crossplatform CI check and I believe we usually pull dependencies via `rosdep`.
If this is necessary for your workflow, please give it an an appropriate name `ci_cross_platform_env.yml`.

Reply:
We are now using a much more minimal `ci_cross_platform_env.yml` and install the dependencies using rosdep (on both Windows and Unix) as suggested.